go.mod

```diff
@@ -3,16 +3,15 @@ module github.com/peggyjv/gravity-bridge
 go 1.15
 
 require (
-	github.com/cosmos/cosmos-sdk v0.44.5
```

Comment:
Since there have been releases since this was opened, we could move this to `v0.45.6`.